src/components/reviewMatches/Dialog.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Dialog as MuiDialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Typography
+} from '@mui/material'
+
+interface DialogProps {
+  buttons: JSX.Element[]
+  title: string
+  subTitle?: string
+  content: JSX.Element | string
+  onOpen: boolean
+  onClose: () => void
+  fullWidth?: boolean
+  maxWidth?: 'sm' | 'md' | 'lg'
+}
+
+const Dialog: React.FC<DialogProps> = ({
+  buttons,
+  title,
+  subTitle,
+  content,
+  onOpen,
+  onClose,
+  fullWidth = false,
+  maxWidth = 'sm'
+}) => {
+  return (
+    <MuiDialog
+      open={onOpen}
+      onClose={onClose}
+      fullWidth={fullWidth}
+      maxWidth={maxWidth}
+    >
+      <DialogTitle>{title}</DialogTitle>
+
+      <DialogContent>
+        <Typography>{subTitle ? subTitle : ''}</Typography>
+        {content}
+      </DialogContent>
+      <DialogActions>{buttons.map(element => element)}</DialogActions>
```

Comment:
```suggestion
      <DialogActions>{buttons}</DialogActions>
```
map does not have any effect.